examples/ola-streaming-client.cpp

```diff
@@ -73,18 +79,37 @@ int main(int argc, char *argv[]) {
   StreamingClient ola_client;
   if (!ola_client.Setup()) {
     OLA_FATAL << "Setup failed";
-    exit(1);
+    exit(ola::EXIT_SOFTWARE);
   }
 
   if (FLAGS_dmx.str().empty()) {
     string input;
+    bool have_universe = false;
+    unsigned int read_universe = FLAGS_universe;
```

Comment:
I wonder if this should be universe or send_universe given it's both the read and the CLI arg universe?

Reply:
Eh.. I think that's fine either way? There's no way to confuse that variable for anything else.

Reply:
It would be nice if it was called something relevant though the flag hasn't really been read from anywhere in that sense.